packages/bits/angular.json

```diff
@@ -99,7 +99,8 @@
               "src/component-styles.less",
               "src/styles/nui-framework-theme.less",
               "src/styles/cdk-overlay-override.less",
-              "demo/src/demostyles.less"
+              "demo/src/demostyles.less",
+              "dist/bundles/css/styles-font-roboto.css"
```

Comment:
If consumers need to do this, do you mind updating the docs to include a code snippet at around line 21 of the docs page below?

Reply:
Not sure what docs page you are meaning?